README.md

```diff
@@ -1,5 +1,5 @@
 # React Pagination
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_pagination/)
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://Bohdan-Dubravin.github.io/react_pagination/)
```

Comment:
https://user-images.githubusercontent.com/38065883/175110324-efd0c986-97a1-454a-b18c-9c0fa267e959.mov

Still a lot of problems here:
- 0 as current page after amount change
- `-4 - 0` as showing items
- NaNs appearing